x-pack/plugin/esql/qa/testFixtures/src/main/resources/row.csv-spec

```diff
@@ -69,13 +69,6 @@ a:integer | b:integer | c:null | z:integer
 1 | 2 | null | null
 ;
 
-evalRowWithNull2
-row a = 1, null, b = 2, c = null, null | eval z = a+b;
```

Comment:
This one had ambiguous attribute names.

Reply:
Should we consider this a bug fix or a breaking change? (IMHO it's a bug, but I see it's questionable)

Reply:
I think yes, this is a bug, since there's no way to refer to the attributes named `null` in following commands.